apps/vanilla/src/script/init.ts

```diff
@@ -0,0 +1,24 @@
+import { SubmitHandler } from '../namespaces/submitHandler';
+
+import { renderUserData } from './renderToUI';
+
+/** Init event for login form. */
+export function initLoginForm(): void {
+  const form = document.querySelector('.form');
+  if (form !== null) {
+    (form as HTMLFormElement).addEventListener('submit', SubmitHandler.submitLoginForm);
+  }
+}
+
+/** Init event for register form. */
+export function initRegisterForm(): void {
+  const form = document.querySelector('.form');
```

Comment:
Please use generic here to avoid casting
```suggestion
  const form = document.querySelector<HTMLFormElement>('.form');
```